ROADMAP.md

```diff
@@ -1,6 +1,50 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.5 Release, Due: March 2022
+## Kubeflow 1.6 Release, Due: July 2022
+* Kubeflow 1.6 [milestones and timelines](https://github.com/kubeflow/community/pull/558)
+
+#### Themes
+* Kubernetes 1.22 support
+
+### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)
+
+#### Kubeflow Pipelines
+* V2 Preview: Support running pipeline in Argo-agnostic approach
+* New DAG visualization based on Pipeline Template and MLMD
+* SDK to change from `kfp.v2` to `kfp`
+* KFP SDK CLI design
```

Comment:
Please add a link to this item: https://docs.google.com/document/d/1Z79ebKaf3hjyadSRsme1Qo7F-Tqu4qM1cyKuLeu2PgA/edit#heading=h.x9snb54sjlu9

Reply:
added